src/pingdb/pingdb_test.go

```diff
@@ -0,0 +1,160 @@
+package main_test
+
+import (
+	"os/exec"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+	"pingdb/internal/testing/docker"
+)
+
+var _ = Describe("PingDB", func() {
+	Context("setup and launch conditions", func() {
+		It("requires a PORT environment variable", func() {
+			cmd := exec.Command(pingdbPath)
+			cmd.Env = append(cmd.Env, "TIMEOUT=1")
+			session, err := gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
+			Expect(err).NotTo(HaveOccurred())
+
+			Eventually(session).Should(gexec.Exit())
```

Comment:
No timeout, so this defaults to 1s.   I would expect these tests to be flaky in CI when containers are overloaded.  

I ran these tests locally against my Podman environment and I see sporadic timeouts as it is.   With 2s this seems to go away (on my machine).  Again, CI is going to be more flaky.

For tests not expecting a timeout errors, I would use a liberal timeout value of several seconds (maybe 30s)

For the test that explicitly test a timeout, a smaller value may be valuable, but I would probably use a "exit" timeout of 10 to 20 seconds and assert the timeout error reports roughly the expected value.